changelog.asc

```diff
@@ -2,6 +2,40 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.20.0 (TBC)
+
+In 2020, implementing Anki's latest scheduling improvements would have taken years. Today, the same process takes weeks due to the extensive effort of merging Anki's codebase into AnkiDroid. Enjoy your even more efficient reviews!
+
+* Your https://opencollective.com/ankidroid/contribute[donations] paid for the work to make this happen so quickly 🤗
+* *AnkiDroid Updates*
+** Includes Anki 24.11, with FSRS 5.0 (1.4.3)
```

Comment:
We should link Anki and FSRS

* Unsure on Anki, https://apps.ankiweb.net/ ?
* https://github.com/open-spaced-repetition/fsrs4anki

Reply:
We should follow the style in the previous announcement: 

[FSRS 4.5](https://github.com/open-spaced-repetition/fsrs4anki) [lib 0.6.4]

Reply:
apps.ankiweb.net sounds fine
I think just get rid of it, lib is only for disambiguation in support and we'll get debug info pasted for that

both IMHO

Reply:
```suggestion
** Includes https://apps.ankiweb.net/[Anki 24.11], with https://github.com/open-spaced-repetition/fsrs4anki[FSRS 5.0]
```